go-controller/pkg/clustermanager/endpointslice_mirror_controller_test.go

```diff
@@ -0,0 +1,442 @@
+package clustermanager
```

Comment:
wrong folder probably :) I suppose this should go to the endpointslicemirror folder.

Reply:
This was kind of intentional, the controller is started through the clustermanager controller so the folder is appropriate here. We use the same approach for egressservices.

Reply:
This is not amazing, because when you run the cluster manager tests these will also be run. I probably don't want to also check this when checking the clustermanager's functionality ...

Plus, you get access to all the private functions on cluster manager, and don't get access to the endpoint slice mirror thingy private functions... 

I share @ricky-rav oppinion that this belongs on the `endpointslicemirror` pkg.

Reply:
You guys convinced me!
Will do.